src/ripple/app/tx/impl/NFTokenCreateOffer.cpp

```diff
@@ -165,11 +165,42 @@ NFTokenCreateOffer::preclaim(PreclaimContext const& ctx)
             return tecUNFUNDED_OFFER;
     }
 
-    // If a destination is specified, the destination must already be in
-    // the ledger.
-    if (auto const destination = ctx.tx[~sfDestination];
-        destination && !ctx.view.exists(keylet::account(*destination)))
-        return tecNO_DST;
+    if (auto const destination = ctx.tx[~sfDestination])
+    {
+        // If a destination is specified, the destination must already be in
+        // the ledger.
+        auto const sleDst = ctx.view.read(keylet::account(*destination));
+
+        if (!sleDst)
+            return tecNO_DST;
+
+        // check if the destination has disallowed incoming offers
+        if (ctx.view.rules().enabled(featureDisallowIncoming))
+        {
+            // flag cannot be set unless amendment is enabled but
+            // out of an abundance of caution check anyway
+
+            if (sleDst->getFlags() & lsfDisallowIncomingNFTOffer)
```

Comment:
I know this is merged already, but should `lsfDisallowIncomingNFTOffer` be `lsfDisallowIncomingNFTokenOffer` to be consistent with the naming convention we established?

Reply:
@kennyzlei @ledhed2222 - thoughts?

Reply:
yeah agreed, I think `NFToken` would be more consistent with the rest of the code than `NFT`

Reply:
@RichardAH any thoughts on this as the original author?  I think someone can help rename since a release has not happened yet since this change was merged

Reply:
@RichardAH has confirmed (directly to me) that we should feel free to rename it

Reply:
Renamed by #4442